src/lib/callApi.js

```diff
@@ -59,11 +72,11 @@ export const getLocalData = (query = undefined) => {
  * @return Promise of all Users data where User username match with query
  * @param query string
  */
-export const getUsersPromise = (query = '') => {
+export const getUsersPromise = (query = '', isSearcAtStart = false) => {
   // local data
   if (SOURCE === 'dev') {
     return new Promise((resolve, reject) => {
```

Comment:
We shouldn't need to wrap this in a `Promise` here - `getLocalData` should handle that internally and return a `Promise` (could be as simple as adding the `async` modifier to the function declaration).

That way you could have a declaration at the top of this file:

```
const queryFxn = SOURCE === 'dev' ? getLocalData : getUsersPromise;
```

And not need to multiple lines of custom code down here.